### DISCUSSION TITLE
Asset Tags search?

### DISCUSSION THREAD

**Discussion by Lucas-Gruber** [directus/directus#7417] [Archive]
Hello, how can I search for an image by its tags in an image or a WYSISYG field? For example, I have a picture of a beach and this picture has the tag "Vacation" but if I enter "Vacation" in the search bar it is not found.

Can anyone help me?

**Comment by rijkvanzanten** [comment]
What database are you using? It could be that it's not found, as the tags are stored as JSON, which can't be searched using a LIKE search in some database types

**Reply by Lucas-Gruber** [reply]
I am using PostgreSQL. Is that possible with Postgres? And if not, is there a way to do it anyway?

**Reply by rijkvanzanten** [reply]
The search bar currently uses a `%LIKE%` search in SQL, which AFAIK isn't available on JSON types in Postgres 🤔

**Reply by Lucas-Gruber** [reply]
Which DB has this support?

**Reply by rijkvanzanten** [reply]
%LIKE% is not an operator for JSON types in most modern databases. The best approach is upvoting https://github.com/directus/directus/discussions/7277 and keep the discussion going around how to do proper in-JSON searching/filtering in the API
